### PR TITLE
PXC-3759 Documentation: gcache.recover provider option is not documen…

### PR DESCRIPTION
**Pull request by patrickbirch**
…ted (5.7)

modified:   source/wsrep-provider-index.rst

### CODE REVIEW

File: doc/source/wsrep-provider-index.rst [hunk 1]
```diff
@@ -432,11 +432,29 @@ size of the disk. Pages are prefixed by gcache.page.
...
+   Percona Database Performance Blog: All You Need to Know About GCache (Galera-Cache)
       https://www.percona.com/blog/2016/11/16/all-you-need-to-know-about-gcache-galera-cache/
 
+.. variable:: gcache.recover
+
+   :cli: No
+   :conf: Yes
+   :scope: Global
+   :dyn: No
+   default: No
```
Comment (nastena1606) [inline doc/source/wsrep-provider-index.rst]: Should it be :default: perhaps?

Comment (patrickbirch) [reply doc/source/wsrep-provider-index.rst]: Fixed.

**Review by alina-derkach-oaza — APPROVED** [review]

File: doc/source/wsrep-provider-index.rst [hunk 2]
```diff
@@ -432,11 +432,29 @@ size of the disk. Pages are prefixed by gcache.page.
...
+.. variable:: gcache.recover
+
+   :cli: No
+   :conf: Yes
+   :scope: Global
+   :dyn: No
+   :default: No
+
+Attempts to recover a node's gcache file to a usable state on startup. If the node can successfully
+recover the gcache file, the node can provide IST to the remaining nodes. This ability can
```
Comment (nastena1606) [inline doc/source/wsrep-provider-index.rst:449]: I would decipher the abbreviation (or at least create the :abbr: directive to give the full name on hover)

Comment (patrickbirch) [reply doc/source/wsrep-provider-index.rst:449]: Noted.

**Pull request merged**